### PR TITLE
fix: studio rls banner warning

### PR DESCRIPTION
**Pull request by Hallidayo**
## What kind of change does this PR introduce?

Supabase Studio > Table Example > RLS Banner

## What is the current behavior?

The banner text for RLS warnings is hard to read:

Screenshot 2023-03-29 at 21 16 24

## What is the new behavior?

Banner text is now easier to read:

Screenshot 2023-03-29 at 21 15 39

### CODE REVIEW

File: studio/components/grid/components/header/RLSBannerWarning.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function RLSBannerWarning() {
     <>
       {!isAcknowledged && !rlsEnabled && isPublicTable ? (
         <div>
-          <div className="text-center bg-amber-500 text-amber-900 text-xs py-2.5 flex items-center justify-center relative">
+          <div className="text-center bg-amber-500 text-scale-1200 text-xs py-2.5 flex items-center justify-center relative">
```
Comment (MildTomato) [inline studio/components/grid/components/header/RLSBannerWarning.tsx:32]: you can try `text-amber-1000` \ `text-amber-1100`

they will still be amber, but easier to read

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: studio/components/grid/components/header/RLSBannerWarning.tsx (hunk 1 above)
Comment (MildTomato) [reply studio/components/grid/components/header/RLSBannerWarning.tsx:32]: might need `"text-amber-1000 dark:text-amber-900"`

**Review by MildTomato — DISMISSED** [review]

File: studio/components/grid/components/header/RLSBannerWarning.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default function RLSBannerWarning() {
             <div className="ml-20 absolute right-2">
               <Button
                 type="outline"
-                className="text-white hover:text-scale-1200"
```
Comment (MildTomato) [inline studio/components/grid/components/header/RLSBannerWarning.tsx:44]: this might be wrong on dark mode

**Review by kiwicopple — APPROVED** [review]

File: studio/components/grid/components/header/RLSBannerWarning.tsx (hunk 2 above)
Comment (kiwicopple) [reply studio/components/grid/components/header/RLSBannerWarning.tsx:44]: @Hallidayo would you mind taking a screenshot of the dark mode too?

**Pull request merged**